supreme_spoon/lp/__init__.py

```diff
@@ -52,6 +52,13 @@ def lower(it):
             return {"vardecl": {"name": id, "value": lower(value)}}
         case ast.Name(id=name, ctx=ast.Load()):
             return {"varref": {"name": name, "type": jl.Type.int64}}
+        case ast.Delete(targets=[ast.Name(id=name, ctx=ast.Del())]):
+            return {
+                "funccall": {
+                    "name": "del",
+                    "args": [{"const": {"value": name, "type": jl.Type.str}}],
+                }
+            }
```

Comment:
Don't disagree with this, and truthfully this is a bit of a hack. I filed an issue in the `midlang` repo to get support for adding a `delete variable` statement - https://github.com/jbirddog/midlang/issues/50

Going to defer until that issue is fixed. Since I am the sole developer of both projects I'll update this repo while testing the fix in `midlang` via the submodules.